### ISSUE TITLE
Erase\write\read stops on random address

### ISSUE THREAD

**Issue by sourenaraya** [urjaman/frser-duino#3]
Hardware: Arduino Duemilanove (FTDI).
Software: flashrom v0.9.8-rc1-r1883, frser-duino from this repo

With original serprog-duino everything works.

**Comment by urjaman** [comment]
Curious. Mind testing with https://github.com/urjaman/flashrom (the default serprog-improvements branch) ?
Maybe use -o logfile.txt with that and pastebin (/gist ...) it.

EDIT: Not that the externally visible behaviour should be any different but the improvements branch has a tiny bit more debug capabilities (and the serprog driver makes better use of the device...).

I tested my Arduino-compatible PTH kit with an FTDI (and my LPC+SPI shield) with the newest frser-duino at 2Mbaud with both r1883 and the new flashrom and noticed no significant difference, 
so i couldnt reproduce this...
(Note: i only have one chip to test with - an 1Mbyte Eon EN25F80)

Usually the freezing is due to somehow a byte being lost in either direction (a command or a reply) and flashrom will just wait for the reply forever (this is a known TODO to atleast not freeze...). 

Lost byte can be due to (list propably not complete):
- the usb-serial converter having a buffer overflow (FTDI so should be fine - unless the usb bus is saturated but that'd be hard to imagine... unless you have some other high-bandwidth usb 1.1 device in use in the same hub)
- the AVR serial buffer being overflown (new flashrom is good about this and frser-duino actually has a buffer unlike serprog-duino that just hoped for being fast enough, so shouldnt happen)
- the AVR being reset for some reason (bad power supply or similar)

So in order to think about doing something about this I'd need logs and details about the setup, and still no promises (I tried the closest setup i had and found no issue...)
